packages/recommend-react/src/__tests__/useFrequentlyBoughtTogether.test.ts

```diff
@@ -24,11 +30,11 @@ function createMockedRecommendClient() {
 }
 
 describe('useFrequentlyBoughtTogether', () => {
-  test('gets Frequently Bought Together products', () => {
+  test('gets Frequently Bought Together products', async () => {
```

Comment:
Let's try to use the Recommend vocabulary consistently.

```suggestion
  test('returns FBT recommendations', async () => {
```